src/components/Dropdown.jsx

```diff
@@ -30,9 +30,10 @@ class Box extends React.Component {
 
     _handleClickOutside(event) {
         const dropdown_dom_node = ReactDOM.findDOMNode(this)
+
         if (!dropdown_dom_node.contains(event.target))
```

Comment:
@henningmu shouldn't this condition be enough to prevent the dropdown from closing when click are inside the dropdown?